app/components/login-component/login/index.hbs

```diff
@@ -0,0 +1,25 @@
+<form ...attributes onsubmit="return false">
+  <div class="{{local-class 'input-group' from='irene/components/login-component/index.scss'}}">
```

Comment:
I think, we could have the file path in a prop so that it could be easy to manage, may be in future.

Reply:
it will be a way more harder to manage if it was abstracted out, since the styles are not a single unit of computation rather are individual on its own.